core/src/main/scala/spinal/core/internals/PhaseVerilog.scala

```diff
@@ -109,26 +111,27 @@ class PhaseVerilog(pc: PhaseContext, report: SpinalReport[_]) extends PhaseMisc
   }
 
   val allocateAlgoIncrementaleBase = globalData.allocateAlgoIncrementale()
-  val usedDefinitionNames = mutable.HashSet[String]()
+  val usedDefinitionNames          = mutable.HashSet[String]()
 
   def compile(component: Component): () => String = {
     val componentBuilderVerilog = new ComponentEmitterVerilog(
-      c                           = component,
-      systemVerilog               = pc.config.isSystemVerilog,
-      verilogBase                 = this,
-      algoIdIncrementalBase       = allocateAlgoIncrementaleBase,
-      mergeAsyncProcess           = config.mergeAsyncProcess,
-      asyncResetCombSensitivity   = config.asyncResetCombSensitivity,
-      anonymSignalPrefix          = if(pc.config.anonymSignalUniqueness) globalData.anonymSignalPrefix + "_" + component.definitionName else globalData.anonymSignalPrefix,
-      nativeRom                   = config.inlineRom,
-      nativeRomFilePrefix         = targetPath,
-      emitedComponentRef          = emitedComponentRef,
-      emitedRtlSourcesPath        = report.generatedSourcesPaths,
-      spinalConfig                = pc.config,
-      pc                          = pc
+      c = component,
+      systemVerilog = pc.config.isSystemVerilog,
+      verilogBase = this,
+      algoIdIncrementalBase = allocateAlgoIncrementaleBase,
+      mergeAsyncProcess = config.mergeAsyncProcess,
+      asyncResetCombSensitivity = config.asyncResetCombSensitivity,
+      anonymSignalPrefix =
+        if (pc.config.anonymSignalUniqueness) globalData.anonymSignalPrefix + "_" + component.definitionName else globalData.anonymSignalPrefix,
+      nativeRom = config.inlineRom,
+      nativeRomFilePrefix = targetPath,
+      emitedComponentRef = emitedComponentRef,
+      emitedRtlSourcesPath = report.generatedSourcesPaths,
+      spinalConfig = pc.config,
+      pc = pc
```

Comment:
Seems the alignement trick didn't worked here, would there be an option for this case ?

Reply:
Yes, this can be customized. See my commits below.